### PR TITLE
CI: Bump imagemagick versions

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       - run:
           name: setup linux
           command: |
-            IMAGEMAGICK_VERSION=6.9.10-35 bash before_install_linux.sh
+            IMAGEMAGICK_VERSION=6.9.10-38 bash before_install_linux.sh
```
Comment (dlemstra) [inline .circleci/config.yml]: This should be changed to 39 that has just been released.

File: .travis.yml [hunk 2]
```diff
@@ -16,8 +16,10 @@ env:
   - IMAGEMAGICK_VERSION=6.7.7-10 CONFIGURE_OPTIONS=--enable-hdri
   - IMAGEMAGICK_VERSION=6.8.9-10
   - IMAGEMAGICK_VERSION=6.8.9-10 CONFIGURE_OPTIONS=--enable-hdri
-  - IMAGEMAGICK_VERSION=6.9.10-35
-  - IMAGEMAGICK_VERSION=6.9.10-35 CONFIGURE_OPTIONS=--enable-hdri
+  - IMAGEMAGICK_VERSION=6.9.9-51
```
Comment (dlemstra) [inline .travis.yml]: We really don't need to add an extra ImageMagick version.

File: .circleci/config.yml (hunk 1 above)
Comment (J0WI) [reply .circleci/config.yml]: They are not yet available on https://ftp.icm.edu.pl/pub/graphics/ImageMagick/binaries/

File: .travis.yml (hunk 2 above)
Comment (J0WI) [reply .travis.yml]: Done

**Pull request merged**